test/testsuites/examples/pod-template.yaml

```diff
@@ -0,0 +1,2 @@
+nodeSelector:
```

Comment:
What's this?

Reply:
This is the pod template used for the tekton CLI. Including it here as an example so it's easily used.